src/escalus_ct.erl

```diff
@@ -86,3 +97,17 @@ is_ct_available() ->
         _ ->
             false
     end.
+
+-spec do_log_stanza(binary(), in | out, exml:element()) -> ok.
+do_log_stanza(Jid, Direction, Stanza) ->
+    ReportString = io_lib:format("~s ~p", [Jid, Direction]),
+    PrettyStanza =
+        try
+            binary:list_to_bin(exml:to_pretty_iolist(Stanza))
```

Comment:
Since this function does exactly the same as `erlang:list_to_binary/1`, I'd change this line to simply `list_to_binary`, but that's only a cosmetic change.